src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
```

Comment:
Consider using React import instead of Component for consistency with the rest of the codebase.